app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -67,39 +67,6 @@
       width: $one-quarter;
     }
 
-    .name-title,
```

Comment:
I feel like once these Whitehall swaps are done more of an intentional sweep of removing redundant styles would be good.